apps/fishing-map/features/dataviews/dataviews.slice.ts

```diff
@@ -388,6 +388,11 @@ export const selectActiveVesselsDataviews = createDeepEqualSelector(
   (dataviews) => dataviews?.filter((d) => d.config?.visible)
 )
 
+export const selectVesselProfileDataview = createDeepEqualSelector(
+  [selectActiveVesselsDataviews, selectVesselInfoData],
```

Comment:
We can grab the vessel id from the URL so we don't need to wait the API info response to resolve the dataview

```suggestion
  [selectActiveVesselsDataviews, selectVesselId],
```